src/main/java/sa/assertj/iterables/AreNot.java

```diff
@@ -0,0 +1,37 @@
+package sa.assertj.iterables;
+
+import org.assertj.core.api.Condition;
+import sa.assertj.Experiment;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AreNot extends Experiment {
+    static Experiment.DataProvider provider = (size, numOfSamples) -> {
+        Object[][] result = new Object[numOfSamples][];
+        for (int i = 0; i < numOfSamples; i++) {
+            ArrayList<Integer> list = new ArrayList<>();
+            Random rand = new Random();
+            for (int j = 0; j < size; j++) {
+                int element = rand.nextInt();
+                if (element == 0) {
+                    element++;
```

Comment:
Then it's always gonna be 1, right?

We can also use the `rand.nextInt(Integer.MAX_VALUE)` here and then check that none are smaller than zero (since it generates from 0 upwards).